libs/build_dod_dataset.py

```diff
@@ -139,7 +176,7 @@ def get_projections_df():
     # "People Tested",
 ]
 
-county_output_cols = ["Province/State",
+our_output_cols = ["Province/State",
```

Comment:
nit: what is our?  i.e.
```suggestion
api_output_cols = ["Province/State",
```